### ISSUE TITLE
Deal with different kernel names

### ISSUE THREAD

**Issue by scottyhq** [xarray-contrib/xarray-tutorial#78]
CI assumes "name": "python3" so if people are using nb_conda_kernels locally or some other kernel name, building the book fails with "jupyter_client.kernelspec.NoSuchKernel: No such kernel named conda-env-xarray-tutorial-py" (https://github.com/xarray-contrib/xarray-tutorial/pull/76#pullrequestreview-1009721167, https://github.com/xarray-contrib/xarray-tutorial/pull/74#issuecomment-1156984097)

For compatibility with binder we want a standard "name":"python3" for all notebooks, so could add a check to https://github.com/xarray-contrib/xarray-tutorial/blob/main/.github/workflows/qaqc.yaml

**Comment by scottyhq** [comment]
Relevant issue: https://github.com/executablebooks/jupyter-book/issues/1287

Would be easy to add some sort of simple check: 
```bash
KERNEL=`jq '.metadata.kernelspec.name' notebook.ipynb`
if [[ $KERNEL != "python3" ]]; then echo 'fix kernel'; fi
```

**Comment by scottyhq** [comment]
closed by #83
